### PR TITLE
Add vue + vite sample

### PR DESCRIPTION
**Pull request by yyx990803**
Main changes in config:

- Use `build --watch` during dev
- Avoid minification during dev
- Disable code splitting and chunk hashing

### CODE REVIEW

**Comment by yyx990803** [comment]
另外建议文档里默认推荐基于 Vite 的配置，因为以后 Vue 3 的默认推荐工具将从 Vue-CLI 转到 Vite。

**Comment by 0x1074** [comment]
Is svelte going to be supported in the future? Thanks!

**Comment by yyx990803** [comment]
@0x1074 both Kraken and Vite are framework agnostic. It's trivial to adapt this example to use Svelte, React, or any framework that works with Vite.

File: demos/hello-vue-vite/index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<script type="module" src="/src/main.js"></script>
```
Comment (cnryb) [inline demos/hello-vue-vite/index.html:1]: src 目录忘记提交了吧

**Review by cnryb — CHANGES_REQUESTED** [review]

**Comment by answershuto** [comment]
image

跑了一下发现似乎没有 src 目录。

**Comment by yyx990803** [comment]
copy 的时候忘记了... 😅

**Review by cnryb — APPROVED** [review]

**Comment by answershuto** [comment]
lgtm

**Comment by andycall** [comment]
我们会在后面支持 ES6 Modules 来提供更好的 vite 开发体验

**Pull request merged**